09-intermediate-rails/session.md

```diff
@@ -108,6 +108,7 @@ Get the most recent version of the book site code from your instructor, then wor
     ```
     </details>
 1. Create the `users#logout` action. This should set `session[:user_id]` to `nil` and redirect the user back to the `root_path`.
+1. Now that you have your routes and controlle actions, update your view(s) so that a user can perform the actions of logging in and logging out.
```

Comment:
```suggestion
1. Now that you have your routes and controller actions, update your view(s) so that a user can perform the actions of logging in and logging out.
```